radix-common/src/crypto/signature_validator.rs

```diff
@@ -22,6 +22,28 @@ pub fn verify_and_recover_secp256k1(
     None
 }
 
+#[cfg(feature = "secp256k1_sign_and_validate")]
+pub fn verify_and_recover_secp256k1_uncompressed(
+    signed_hash: &Hash,
+    signature: &Secp256k1Signature,
+) -> Option<[u8; secp256k1::constants::UNCOMPRESSED_PUBLIC_KEY_SIZE]> {
```

Comment:
Is it worth us adding an explicit new type for this, e.g. `ExpandedSecp256k1PublicKey` - which we can use here and in scrypto? (And can be a semantic transparent wrapper around a `pub [u8; 65]`)

Reply:
And maybe adding to the Rust doc for `Secp256k1PublicKey` that it's the compressed public key, which is the default format used in the Radix stack?

Reply:
I will add the wrapper.